### PR TITLE
HPCC-22150 Document Std.File functions DFUServerQueue option

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@AttilaVamos Please review (there is a link to the PDF below)

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

http://10.240.32.243/view/Docs%207.0/job/JimDocBuild7/CE=ubuntu-18.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/EN_US/ECLStandardLibraryReference_EN_US-7.4.1-closedown0.pdf#page=62

### CODE REVIEW

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml [hunk 1]
```diff
@@ -240,6 +245,13 @@
           ascii.</entry>
         </row>
 
+        <row>
+          <entry><emphasis>dfuServerQueue</emphasis></entry>
+
+          <entry>Name of target DFU Server queue. Default is ' ' (blank) for
```
Comment (AttilaVamos) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayVariable.xml]: Same as above

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayXML.xml [hunk 2]
```diff
@@ -182,6 +186,13 @@
           -1 (never expires).</entry>
         </row>
 
+        <row>
+          <entry><emphasis>dfuServerQueue</emphasis></entry>
+
+          <entry>Name of target DFU Server queue. Default is ' ' (blank) for
```
Comment (AttilaVamos) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayXML.xml]: Same as above.

File: docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayFixed.xml [hunk 3]
```diff
@@ -157,6 +161,13 @@
           -1 (never expires).</entry>
         </row>
 
+        <row>
+          <entry><emphasis>dfuServerQueue</emphasis></entry>
+
+          <entry>Name of target DFU Server queue. Default is ' ' (blank) for
```
Comment (AttilaVamos) [inline docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayFixed.xml]: By the source code:
```
const char * dfuServerQueue=nullptr
```
it is empty '' rater than ' ' (blank)

**Review by AttilaVamos — CHANGES_REQUESTED** [review]
@JamesDeFabia back to you.

**Comment by AttilaVamos** [comment]
@JamesDeFabia it seems the sequence of my comments is a bit out of order (the first became the last).

**Comment by JamesDeFabia** [comment]
@AttilaVamos I made the edits you requested (and I edited the JavaDoc comments in **file.ecl**, as well). Please have another look.

**Review by AttilaVamos — APPROVED** [review]
@JamesDeFabia It seems good.

**Comment by JamesDeFabia** [comment]
@richardkchapman This is now good to merge

**Pull request merged**